ui/package.json

```diff
@@ -56,15 +56,17 @@
     ]
   },
   "devDependencies": {
+    "@testing-library/react-hooks": "^5.0.3",
     "@types/react": "^16.9.49",
     "@typescript-eslint/eslint-plugin": "4.0.1",
     "@typescript-eslint/parser": "4.0.1",
+    "enzyme": "^3.11.0",
```

Comment:
I see the version of the packages are similar then why are these changes done :thinking:

Reply:
I guess these changes are done  by `npm i` to sort packages

Reply:
I don't think it should, there's only one new package added